src/spatial_reaction_systems/spatial_reactions.jl

```diff
@@ -95,7 +95,7 @@ function check_spatial_reaction_validity(rs::ReactionSystem, tr::TransportReacti
     if any([isequal(tr.species, s) && !isequivalent(tr.species, s) for s in species(rs)]) 
         error("A transport reaction used a species, $(tr.species), with metadata not matching its lattice reaction system. Please fetch this species from the reaction system and used in transport reaction creation.")
     end
-    if any([isequal(rs_p, tr_p) && !equivalent_metadata(rs_p, tr_p) 
+    if any([isequal(rs_p, tr_p) && !isequivalent(rs_p, tr_p) 
```

Comment:
Previously parameters could only differ in metadata. Now they can differ in type as well.

Reply:
No, that should probably be avoidable, will update.

Reply:
Updated. I think these ones were written before I knew that this was something you could do